python/cuml/test/test_fil.py

```diff
@@ -425,29 +425,27 @@ def test_lightgbm(tmp_path, num_classes):
                  'metric': 'multi_logloss',
                  'num_class': num_classes}
     num_round = 5
+    model_path = str(os.path.join(tmp_path, 'lgb.model'))
+
     bst = lgb.train(param, train_data, num_round)
-    gbm_preds = bst.predict(X)
-    if num_classes > 2:
-        gbm_preds = gbm_preds.argmax(axis=1)
-    model_path = str(os.path.join(tmp_path,
-                                  'lgb.model'))
     bst.save_model(model_path)
-    fm = ForestInference.load(model_path,
-                              algo='TREE_REORG',
-                              output_class=True,
-                              model_type="lightgbm")
-    fil_preds = fm.predict(X)
```

Comment:
The current code tests both `predict()` and `predict_proba()` from FIL, whereas the new code only tests `predict_proba()`. What is the performance gain from not testing `predict()`?

Reply:
The new code calls `predict_proba()` for binary case and `predict()` for the multi-class case. Should we test both `predict_proba()` and `predict()` for the binary classifier? The performance impact is likely to be minor, since most perf gain comes from not fitting LightGBM model twice.

Reply:
We should test both `predict()` and `predict_proba()` in the binary case. For the multi-class case, it's a good idea to add a TODO to test `predict_proba()` once it is supported.

Reply:
Got it. Will do so.

Reply:
Addressed in my latest commit.